docs/stake-pool-guide/node-installation-process.md

```diff
@@ -0,0 +1,239 @@
+---
+id: node-installation-process
+title: Cardano Node Installation process (Environment Setup)
+sidebar_label: Cardano Node Installation process
+description: Cardano Node Installation process (Environment Setup)
+image: ../img/og-developer-portal.png
+---
+The Cardano node is the core component that underpins the Cardano network. Ultimately, a blockchain network is just a collection of interconnected nodes, all working together to validate transactions and blocks by means of consensus. The definition of consensus for any given network varies, but for the Cardano network it’s defined by the [Ouroboros](stake-pool-basics#understanding-consensus) protocol.
+
+In this section, we will walk you through the process of downloading, compiling, and installing `cardano-node` and `cardano-cli` into your **Linux-based** operating system.
+
```

Comment:
A link & reference needs to be added here, to the most basic (and well maintained) source file for the installation procedure: https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/install.md

As commented above regarding the videos, material can go out of date any time the tutorial stops being monitored.  Eventually that will happen this written material: in fact, very quickly after versions of GHC, Cabal, `cardano-node`, libraries, and their package dependencies change.  So since this material is mostly a re-hash of the official installation procedure, they have to be directed there for help & comparison when necessary.

Reply:
added at the end

Reply:
1. The link added at the end is *not* the official instructions for installing the node.  The link I quoted in the comment above is rigorously and responsibly maintained and is the least likely of any document to _ever_ go out of date.  So unless another editor objects, or you can offer a reason why not, I have to insist that this is the reference link included.
2. This link has to go _at the beginning of the document_, not at the end.  The reason is simple: if and when people get into problems months & years down the road, although they may still find this material useful they will need to know of an alternative source *before* they get into trouble.

I'll move & update this reference & comment in another commit (in a few minutes), so we can consider this issue tentatively resolved unless there are any objections.